.travis.yml

```diff
@@ -39,6 +42,14 @@ env:
 
 before_script:
   - travis_retry composer update ${COMPOSER_FLAGS} --no-interaction --prefer-dist
+  - if [[ $BCMATH == "off" ]]; then
```

Comment:
That's for the master branch only. In the 2.0 branch (which is what the updated composer.json uses) bcmath takes priority:

https://github.com/phpseclib/phpseclib/blob/2.0/phpseclib/Math/BigInteger.php#L253

I could update it to prioritize the PHP engine over bcmath but the 1.0/2.0 branches don't distinguish between PHP32 and PHP64 like the 3.0 and master branches do. Or I could update the logic of the switch statement to look at `PHP_INT_SIZE` and not just at what extensions are loaded!

Reply:
I see it is possible to control phpseclib by defining `MATH_BIGINTEGER_MODE`. Isn't it better approach rather than recompiling things?

Reply:
I think it's a better approach lol but I guess I didn't pursue / recommend that approach initially because, unless you dig through the phpseclib source code, you kinda have to take it on faith that it really isn't using bcmath. If the bcmath extension isn't even available then you don't have to take it on faith at all - you know it's not using bcmath because it isn't possible for it to be using bcmath.

But, that said, the `MATH_BIGINTEGER_MODE` approach would eliminate the lengthy unit tests that re-compilation brings and it'd eliminate the "/home/travis/.travis/functions: line 109: vendor/bin/phpunit: No such file or directory" errors that Docker is intermittently giving me and the other issues associated with the Docker approach!

It's what the phpseclib unit tests do in lieu of recompiling / Docker containers.

Reply:
`MATH_BIGINTEGER_MODE` is now being used, as of the latest commit!